### ISSUE TITLE
R2 explicit statement of best practice

### ISSUE THREAD

**Issue by dagewa** [dagewa/DIALS_for_ED_paper#33]
> The paper is well written, but somewhat too long for my taste when it elaborates on the coupling of refineable parameters. It is good to be able to diagnose the coupling of parameters, but the solution  appears to be e.g.
> - calibrate the distance as well as possible (should be self-evident)
> - don't refine it in DIALS (unless the data are extremely good)
> - refine the cell in REFMAC (and derive the true distance from the result)
> and maybe it would be more helpful to the reader to find this explicitly stated.

**Comment by dagewa** [comment]
Perhaps we can comment in the paper that although distance calibration implies extra work by the experimenter, it is important, and would become essential if samples with unknown cell constants are analysed. We can also make the point that the best protocol we found for _these data_ were to fix distance and use the cell refinement in REFMAC to improve the cell (at that point, we don't really care about the distance. In principle, we could feed it back into data processing but it is doubtful that this would improve any results).

**Comment by dagewa** [comment]
As noted by @tgruene: The procedure suggested for the cell /distance was, simplified, applied in https://doi.org/10.1002/chem.201704213

Seems to be another one I don't have access to :-(

**Comment by cmaxtb** [comment]
Second comment makes sense, lets do that, we can refer to the article

**Comment by tgruene** [comment]
Hi David, your phrasing is reasonable "we found for these data ...". I suggest to go with it.

**Comment by dagewa** [comment]
Reply:
We added a note about the best practice we found for these datasets, with the caveat that must be each situation must be investigated on a case-by-case basis as there is no generalised protocol that applies yet. The diagnostics help to explore the difficult cases, but do not solve the problems by themselves. We also cited the paper by Gruene _et al._ (2018) in which adjustments to the cell were made alongside the comment that this is possible in Refmac (there is no citation available for the option as implemented in Refmac yet). It is not immediately clear which parts on the coupling of parameters could be trimmed without losing meaning, so we have retained that section.

**Comment by tgruene** [comment]
"We also cited the paper by Gruene et al. (2017) in which adjustments to the cell were and add  the comment"
(there is a 'frontispiece' from 2018, which confusingly also got a DOI, but the paper appeared 2017. i also change the 'alongside...'. To me that sounded like Gruene et al(2017) contains the reference to Refmac, instead of here in this paper.

**Comment by dagewa** [comment]
Thanks Tim, I'll fix these

**Comment by dagewa** [comment]
The DOI seems to be the same? The [publication history](https://onlinelibrary.wiley.com/doi/abs/10.1002/chem.201704213) does say "Accepted manuscript online: 28 November 2017", but also "Version of Record online: 17 January 2018". Sites such as [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/29193398) go for 2018. Do you still want to change to 2017?

**Comment by tgruene** [comment]
Hi David,
you are right - it says 'first published Nov. 2017', but the downloaded citation has 2018. Please go with 2018.
Thanks for spotting this, I have to correct my bib-files

**Comment by tgruene** [comment]
It's a good sign - the publisher would push papers, where they expect many citations, to the beginning of the year ;-)

**Comment by dagewa** [comment]
Excellent, I adjust the reply as follow and close the issue:

We added a note about the best practice we found for these datasets, with the caveat that each situation must be investigated on a case-by-case basis, as there is no generalised protocol that applies across the board. The diagnostics we developed help to explore difficult cases, but do not solve the problems by themselves. The improvement of the unit cell at the stage of model refinement is a useful concept, particularly for electron diffraction, that is worth emphasising. We added a citation to the paper by Gruene _et al._ (2018) in which this procedure was also mentioned. It is not immediately clear which parts on the coupling of parameters could be trimmed without losing meaning, so we have retained that section.
